<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GenaratorWindow</class>
 <widget class="QDialog" name="GenaratorWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="titleLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>161</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Passphrases generator</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>360</y>
     <width>83</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="generateButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>360</y>
     <width>83</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Generate</string>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>321</width>
     <height>25</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QLabel" name="inputFileLabel">
    <property name="text">
     <string>Words file:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputPathLine"/>
   <widget class="QPushButton" name="inputFileButton">
    <property name="maximumSize">
     <size>
      <width>31</width>
      <height>25</height>
     </size>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
  </widget>
  <widget class="QSplitter" name="splitter_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>321</width>
     <height>141</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QSplitter" name="splitter_2">
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QLabel" name="includeSymbols">
     <property name="text">
      <string>Symbols:</string>
     </property>
    </widget>
    <widget class="QLabel" name="includeUpperLetters">
     <property name="text">
      <string>Upper Letters:</string>
     </property>
    </widget>
    <widget class="QLabel" name="includeLowerLetters">
     <property name="text">
      <string>Lower Letters:</string>
     </property>
    </widget>
    <widget class="QLabel" name="includeNumbers">
     <property name="text">
      <string>Numbers:</string>
     </property>
    </widget>
    <widget class="QLabel" name="includeAmbigous">
     <property name="text">
      <string>Ambigous:</string>
     </property>
    </widget>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QCheckBox" name="symbolsCheckBox">
     <property name="text">
      <string>!@#$%^&amp;*()-+_=?</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="upperLettersCheckBox">
     <property name="text">
      <string>ABCD...XYZ</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="lowerLettersCheckBox">
     <property name="text">
      <string>abcd...xyz</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="numbersCheckBox">
     <property name="text">
      <string>0123...89</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="ambigousCheckBox">
     <property name="text">
      <string>{ } [ ] ( ) /  ' &quot; ` ~ , ; : . &lt; &gt;</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QSplitter" name="splitter_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>280</y>
     <width>321</width>
     <height>25</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QLabel" name="outputFileLabel">
    <property name="text">
     <string>Output file:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="outputPathLine"/>
   <widget class="QPushButton" name="outputFileButton">
    <property name="maximumSize">
     <size>
      <width>31</width>
      <height>25</height>
     </size>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
